Check/Cache/UsedBackends.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\Cache\Bins.
+ */
+
+class SiteAuditCheckCacheUsedBackends extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Used Backends');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Detail cache backend used by each bin.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {}
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {
+    if (drush_get_option('html')) {
+      $ret_val = '<table class="table table-condensed">';
+      $ret_val .= '<thead><tr><th>Bin</th><th>Class</th></tr></thead>';
+      $ret_val .= '<tbody>';
+      foreach ($this->registry['used_backends'] as $bin => $class) {
+        $ret_val .= "<tr><td>$bin</td><td>$class</td></tr>";
+      }
+      $ret_val .= '</tbody>';
+      $ret_val .= '</table>';
+    }
+    else {
+      $ret_val  = 'Bin: Class' . PHP_EOL;
+      if (!drush_get_option('json')) {
+        $ret_val .= str_repeat(' ', 4);
+      }
+      $ret_val .= '----------';
+      foreach ($this->registry['used_backends'] as $bin => $class) {
+        $ret_val .= PHP_EOL;
+        if (!drush_get_option('json')) {
+          $ret_val .= str_repeat(' ', 4);
+        }
+        $ret_val .= "$bin: $class";
+      }
+    }
+    return $ret_val;
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultPass().
+   */
+  public function getResultPass() {}
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultWarn().
+   */
+  public function getResultWarn() {
+    return dt('Using the database as a caching backend, which is less efficient than a dedicated key-value store.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getAction().
+   */
+  public function getAction() {
+    if ($this->score == SiteAuditCheckAbstract::AUDIT_CHECK_SCORE_WARN) {
+      return dt('Consider using a caching backend such as redis or memcache. For details on how to configure alternative caching backends, see the Configuration section in https://api.drupal.org/api/drupal/core!modules!system!core.api.php/group/cache/8 .');
```

Comment:
Instead of `see the configuration section`, use `see https://api.drupal.org/api/drupal/core%21modules%21system%21core.api.php/group/cache/8#configuration`. If the branch was on this repository, I'd just make the change myself ;-)